### PR TITLE
Drop support for Python 3.8; XBlock <2

### PR DESCRIPTION
**Pull request by mrtmm**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,11 +1,10 @@
 [tox]
-envlist = py{38,311,312},pipdeptree{,-requirements},flake8,report
+envlist = py{311,312}-{pipdeptree,pipdeptree-requirements,flake8},report
```
Comment (fghaas) [inline tox.ini:2]: This way you disable the `manage.py test` tests; that's why the `coverage` reports are empty.

This should read:

```suggestion
envlist = py3{11,12}{,-pipdeptree,-pipdeptree-requirements,-flake8},report
```
```

File: tox.ini [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 
 [gh-actions]
 python =
-    3.8: py38,pipdeptree{,-requirements},flake8
-    3.11: py311,pipdeptree{,-requirements},flake8
-    3.12: py312,pipdeptree{,-requirements},flake8
+    3.11: py311-{pipdeptree,pipdeptree-requirements,flake8}
+    3.12: py312-{pipdeptree,pipdeptree-requirements,flake8}
```
Comment (fghaas) [inline tox.ini:7]: Same problem as above. Should be fixable like this:

```suggestion
    3.11: py311{,-pipdeptree,-pipdeptree-requirements,-flake8}
    3.12: py312{,-pipdeptree,-pipdeptree-requirements,-flake8}
```

File: tox.ini (hunk 1 above)
Comment (mrtmm) [reply tox.ini]: I see, thanks!

**Review by fghaas — APPROVED** [review]
Looks good to me now, thanks!

**Pull request merged**